src/diff/index.test.ts

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+
+import { expect, test } from "@oclif/test";
+import { writeSync } from "fs";
+import { FileResult, fileSync } from "tmp";
+
+import cmd from "../commands/diff";
+
+const createTempFile = (content: string): FileResult => {
+  const tempFile = fileSync();
+  writeSync(tempFile.fd, content);
+  return tempFile;
+};
+
+const ramlOld = createTempFile(
```

Comment:
1) Why are tmp files needed?
2) Maybe consider having these in their own files and copy them to a tmp dir for these tests.

Reply:
These are tests of the CLI and not the diff functions themselves (those have their own tests) so I prefer having these inline since they're meant to be as simple as possible.